config/protobuf-events.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Client Identifier to use as origin in a protobuf communication. Allows to isolate different services
+    |--------------------------------------------------------------------------
+    */
+    'client' => env('RABBITEVENTS_CLIENT_ID', 'laravel-protobuf-events'),
```

Comment:
I rather null default value to avoid clients to be set to this value without knowing it.